pkg/util/common.go

```diff
@@ -0,0 +1,38 @@
+package util
+
+import "strings"
+
+func MergeMaps(ms ...map[string]string) map[string]string {
+	res := map[string]string{}
+	for _, m := range ms {
+		for k, v := range m {
+			res[k] = v
+		}
+	}
+	return res
+}
+
+func KeysFromMap(ms map[string]string) []string {
+	var res []string
+	for k := range ms {
+		res = append(res, k)
+	}
+	return res
+}
+
+func FlattenKey(key string) string {
```

Comment:
nitpick: maybe it is better to call this `FlattenString`

Reply:
Make sense. Making a change.